gandalf/src/main/java/io/github/btkelly/gandalf/models/Bootstrap.java

```diff
@@ -83,4 +85,54 @@ public Bootstrap createFromParcel(Parcel source) {
             return new Bootstrap[size];
         }
     };
+
+    public static class Builder {
```

Comment:
for the builder, how about instead doing
```
builder.setAlert(message, blocking)
           .setOptionalUpdate(message, version)
           .setRequiredUpdate(message, version);
```
might feel a bit less cumbersome, and also make you less likely to miss a value that you should have.